### PR TITLE
Add email to comments

### PR DESCRIPTION
**Pull request by hannahsalameh**
Made it so that the users email will display when they make a comment. I am going to add CSS in the next PR so it looks good, too.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -83,20 +95,21 @@ public long getId(){
 
 @Override
 public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+  UserService userService = UserServiceFactory.getUserService();
```
Comment (ofk-goog) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:98]: nit: indentation does not match

File: portfolio/src/main/java/com/google/sps/servlets/DeleteServlet.java [hunk 2]
```diff
@@ -38,5 +42,6 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
      for (Entity entity : commentList.asIterable()) {
          datastore.delete(entity.getKey());
      }
-}
+   }
```
Comment (ofk-goog) [inline portfolio/src/main/java/com/google/sps/servlets/DeleteServlet.java:45]: nit: It's worth keeping the indentation of code looking good. Which in this case means indenting code that is now within the `if` block. Your text editor can probably do this automatically.

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -12,20 +12,25 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-function displayPage(){
-  fetch('/loginStatus').then(response => response.json()).then(data =>{
-    if(data.loginStatus == false){
-      const body = document.getElementById("content");
-      body.innerHTML = "";
+function loggedInCheck(){
```
Comment (ofk-goog) [inline portfolio/src/main/webapp/script.js:15]: Good, thanks for making the function name more specific

**Review by ofk-goog — APPROVED** [review]
Looks good except a couple superficial formatting issues

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (hannahsalameh) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:98]: wait, which indentation? the Userservice line and the if line below, or the Userservice line and the public void line above?

Comment (ofk-goog) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:98]: One or the other. They should match. Most text editors will have an opinion about this, if you use the auto-format feature

Comment (ofk-goog) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:98]: Oh wait, I misread your question. "UserService" should line up with "Query" on the line below

**Pull request merged**